packages/third-parties/package.json

```diff
@@ -25,7 +25,8 @@
   "devDependencies": {
     "next": "13.4.20-canary.38",
     "outdent": "0.8.0",
-    "prettier": "2.5.1"
+    "prettier": "2.5.1",
+    "rimraf": "^3.0.2"
```

Comment:
No need to add a dependency here.

- See https://github.com/vercel/next.js/pull/55547